guides/onboarding/README.md

```diff
@@ -0,0 +1,9 @@
+Onboarding
+==========
+
+## Table of Contents
+ - [New developer!](developer.md)
```

Comment:
![image](https://user-images.githubusercontent.com/5278175/46071599-11e99080-c189-11e8-9919-e612bf15667f.png)


I prefer to have `Onboarding` as simple and predictable name